libcxx/utils/generate_feature_test_macro_components.py

```diff
@@ -992,6 +992,7 @@ def add_version_header(tc):
             "name": "__cpp_lib_ranges",
             "values": {
                 "c++20": 202207,
+                "c++23": 202211,  # P2602R2 Poison Pills are Too Toxic
```

Comment:
(Not strictly related to this PR) I think the value is fine in C++23 mode, but perhaps it should be `202110L` in C++20 mode as explained in #101715.

Reply:
Ack, we can fix that in a follow-up PR indeed.